# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1524168066 14400
#      Thu Apr 19 16:01:06 2018 -0400
# Branch stable
# Node ID ff59be58f0282e418fb3447d5442fa1d618cfe10
# Parent  f692728de1133dc905ae1eb256c9f933480d1c25
bump version for second 4.4 release candidate

* configure.ac (AC_INIT): Set version to 4.3.91.
(OCTAVE_PATCH_VERSION): Now 91.
(OCTAVE_RELEASE_DATE): Set to 2018-04-19.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,34 +15,34 @@ dnl Copyright (C) 1993-2018 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [4.3.90], [https://octave.org/bugs.html], [octave],
+AC_INIT([GNU Octave], [4.3.91], [https://octave.org/bugs.html], [octave],
         [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=3
-OCTAVE_PATCH_VERSION=90
+OCTAVE_PATCH_VERSION=91
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION argument.
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2018 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2018-04-11"
+OCTAVE_RELEASE_DATE="2018-04-19"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do this is with
 ## library versioning, but not all systems support it.
 ##
 ## NOTE: This macro will be removed in a future version of Octave!
 ## If you insist on checking for features using a version number, use the
